definitions/divorce/json/CaseEvent.json

```diff
@@ -2755,5 +2755,15 @@
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "04/05/2020",
```

Comment:
Is this a suitable date?

Reply:
as long as it's not in the past it's fine